bib/cmd/bootc-image-builder/image.go

```diff
@@ -154,6 +156,195 @@ func pipelinesForDiskImage(c *ManifestConfig, rng *rand.Rand) (image.ImageKind,
 	return img, nil
 }
 
+func pipelinesForISO(c *ManifestConfig, rng *rand.Rand) (image.ImageKind, error) {
+	if c.Imgref == "" {
+		fail("pipeline: no base image defined")
+	}
+	ref := "ostree/1/1/0"
+	tlsVerify := true
+	containerSource := container.SourceSpec{
+		Source:    c.Imgref,
+		Name:      c.Imgref,
+		TLSVerify: &tlsVerify,
+	}
+
+	img := image.NewAnacondaContainerInstaller(containerSource, ref)
+	img.ExtraBasePackages = rpmmd.PackageSet{
+		Include: []string{
+			"anaconda-dracut",
+			"atheros-firmware",
+			"brcmfmac-firmware",
+			"curl",
+			"dracut-config-generic",
+			"dracut-network",
+			"hostname",
+			"iwlwifi-dvm-firmware",
+			"iwlwifi-mvm-firmware",
+			"kernel",
+			"linux-firmware",
+			"less",
+			"nfs-utils",
+			"openssh-clients",
+			"ostree",
+			"plymouth",
+			"realtek-firmware",
+			"rng-tools",
+			"rpcbind",
+			"selinux-policy-targeted",
+			"systemd",
+			"tar",
+			"xfsprogs",
+			"xz",
+		},
+	}
+
+	img.ExtraBasePackages = img.ExtraBasePackages.Append(rpmmd.PackageSet{
```

Comment:
possibly stupid question: Why are the packages split into two groups? (happy to accept it, but I think that adding a comment with the reason would be helpful to future readers)

Reply:
Because they were split in two in `images`, I've consolidated them.